content/03.methods.md

```diff
@@ -117,7 +117,12 @@ Per recommendations by the New York Genome Center [@doi:10.1101/623702], for WGS
 
 #### VCF annotation and MAF creation
 
-We filtered outputs from both callers on the "PASS" filter, and annotated using The ENSEMBL Variant Effect Predictor [@doi:10/gdz75c], reference release 93, and created MAFs using MSKCC's vcf2maf [@url:https://github.com/mskcc/vcf2maf] v1.6.17.
+INDEL renormalization of variants from each caller was performed using the following workflow [@url:https://github.com/d3b-center/OpenPBTA-workflows/blob/master/cwl/norm_annot_patch_wf.cwl].
```

Comment:
not sure about the renormalization wording, but besides the CWL workflow itself, we could also mention about its core method, which is a `bcftools norm`: http://samtools.github.io/bcftools/bcftools.html#norm

Reply:
Also I wonder if we should copy the tools/sub-workflows CWL to complete the `norm_annot_patch_wf.cwl` and some other CWL files under the [cwl folder](https://github.com/d3b-center/OpenPBTA-workflows/blob/master/cwl), because those CWL are just `Workflow` class, usually they refers to other tools and sub-workflows in its steps like this https://github.com/d3b-center/OpenPBTA-workflows/blob/111c4e299c41446053628605f49593a1869b28ba/cwl/norm_annot_patch_wf.cwl#L38

and this https://github.com/d3b-center/OpenPBTA-workflows/blob/111c4e299c41446053628605f49593a1869b28ba/cwl/norm_annot_patch_wf.cwl#L47

so just to have one sole `norm_annot_patch_wf.cwl` won't work if someone wanted to rerun it. 

maybe we can just redirect them its origin CWL repo too.

Reply:
Ok thanks, I'll update!

Reply:
@yuankunzhu, i have updated the text and added the whole subworkflow. Not sure if you wanted me to add the entire tools folder to that repo as well, or certain ones as the one you mentioned?